### PR TITLE
Added logging of tensorflow and pytorch tensors as image

### PR DESCRIPTION
**Pull request by jakubczakon**
* added logging of tensorflow tensor as image
* added logging of pytorch tensor as image
* fixed scaling of logging numpy arrays as image

### CODE REVIEW

**Comment by jakubczakon** [comment]
I have some problems running this on all OS/python version.

Right now the problem is with torch installation (during tests). 
On linux it works fine with

`
pip install torch
`

on windows, however, the proper way to install it is

`
pip install torch -f https://download.pytorch.org/whl/torch_stable.html
`

How should I approach this @pitercl ?

**Comment by pitercl** [comment]
I think we can add the `-f https://...` option here: https://github.com/neptune-ai/neptune-client/blob/master/.github/workflows/neptune-client.yml#L18 

This means that torch will be downloaded from there on all systems, but I don't see that as a big problem. Also, please add a comment there explaining why you're doing so.

**Comment by jakubczakon** [comment]
mhm so drop it from the test_requirements and put it directly in the CI @pitercl ?

**Comment by pitercl** [comment]
I rather meant something like this:
```yaml
run: |
          pip install -r requirements.txt
          pip install -f https://download.pytorch.org/whl/torch_stable.html -r test_requirements.txt

```

**Comment by jakubczakon** [comment]
It might have helped but now I am hitting this pylint error on python 2. 
I think it may be related to this https://github.com/PyCQA/pylint/issues/3312 -> what worries me there is the "drop python2 support Merged" at the bottom of this issue.

I need help to move forward here.

**Comment by jakubczakon** [comment]
Also I just saw that we do have 2 pylint versions in our test_requirements:

```
pylint==1.9.4; python_version < '3.0'
pylint==2.4.2; python_version >= '3.0'
```

I don't know how to move forward here @pitercl

File: neptune/internal/utils/image.py [hunk 1]
```diff
@@ -54,6 +45,22 @@ def get_image_content(image):
         except ImportError:
             pass
 
+        if sys.version_info[0] >= 3:
```
[the remark this answers is not in the document]
Comment (jakubczakon) [reply neptune/internal/utils/image.py:48]: You are right it's redundant -> I'll drop it

File: neptune/internal/utils/image.py [hunk 2]
```diff
@@ -62,7 +69,23 @@ def _get_figure_as_image(figure):
...
 
+
 def _get_pil_image_data(image):
     with io.BytesIO() as image_buffer:
         image.save(image_buffer, format='PNG')
         return image_buffer.getvalue()
+
+
+def _get_numpy_as_image(array):
+    array *= 255
```
[the remark this answers is not in the document]
Comment (jakubczakon) [reply neptune/internal/utils/image.py:80]: Well not exactly. If it was 0-255 and is multiplied again PIL still displays it nicely. 
If it was 0-1 then multiplying by 255 converts it to a range that PIL can handle/display.

**Review by pitercl — APPROVED** [review]

**Pull request merged**